drivers/spi/spi_nrfx_spi.c

```diff
@@ -286,7 +286,7 @@ static int spi_nrfx_pm_action(const struct device *dev,
 
 	case PM_DEVICE_ACTION_SUSPEND:
 		if (data->initialized) {
-			nrfx_spi_uninit(&config->spim);
```

Comment:
Looks like an issue that should get fixed into 3.0, @anangl ?

Reply:
Definitely. Too bad such obvious copy/paste mistake was not caught in the review of #42304 or by CI somewhere... I'll prepare a PR with the fix for it.

Reply:
Should I drop this SPI patch commit?

Reply:
yes, has been fixed